modules/ROOT/Edits/endpoint-https-receive.adoc

```diff
@@ -0,0 +1,107 @@
+= HTTP and HTTPS Receive Endpoint Settings
+
+HTTP and HTTPS receive endpoint settings configure where to receive messages in the following context:
+
+* Source at host
++
+Source endpoint in an outbound message flow that receives application JSON and XML messages from the backend systems
+
+Only a host can own HTTP and HTTPS receive endpoints.
+
+Create HTTP and HTTPS receive endpoints by configuring these settings on the host page:
+
+[%header%autowidth.spread]
+|===
+|Name |Description |Required
+|Endpoint Name
+|Automatically-generated name to identify the endpoint. You can modify this value later on.
+|Yes
+
+| Owner
+| Host that owns the endpoint.
+| Yes
+
+| Protocol
+| Protocol selected in the *Protocol* field of the *New endpoint*. You cannot modify this value.
+| Yes
+
+| Description
+| User-supplied value that describes the purpose and use of the endpoint.
+| No
+
+|URL
+a|HTTP or HTTPS address where you receive outbound source messages from backend systems.
+|Yes
+
+|Response timeout (in milliseconds)
+|Response timeout period in milliseconds. The default is  `15000`.
+|Yes
+
+|Authentication type
+a|Type of authentication method used to protect the HTTP receive service.
+
+|Yes
+
+|Basic authentication
+a|User name and password that protects the HTTP listener.
+
+After saving your new or reconfigured endpoint, you cannot subsequently retrieve or modify the password.
+
+The backend applications that send JSON or XML payloads to the HTTP or HTTPS receive endpoint must pass the credentials in the HTTP request header.
+|Yes
+
+|API Key
+a|API key for the HTTP listener.
+
+After selecting *API Key* in the *Authentication type* list, specify the HTTP header name and API key to protect the HTTP listener. After saving your new or reconfigured endpoint, you cannot subsequently retrieve or modify the API key.
+
+The backend applications that send JSON or XML payloads to the HTTP receive endpoint must pass the API key in the HTTP request with the configured HTTP header name.
+|Yes
+|===
+
+== HTTP or HTTPS Receive Endpoint URL
+
+You can deploy an HTTP or HTTPS receive endpoint on a public port, private port, or on-premises.
```

Comment:
ditto suggestions for "AS2 Receive Endpoint URL" earlier, re. "Deployment" vs. "URL"

Reply:
In fact, please apply all appropriate edits in the "AS2 Receive Endpoint URL" topic set to this one as well, since the texts are parallel.